code/DDSCodeTester.cpp

```diff
@@ -2040,6 +2040,45 @@ void dds_topic_examples()
         }
         //!--
     }
+
+    {
+        //DDS_DYNAMIC_TYPES_IDL_PARSING
+        // Create a DomainParticipant in the desired domain
+        DomainParticipant* participant =
+                DomainParticipantFactory::get_instance()->create_participant(0, PARTICIPANT_QOS_DEFAULT);
+        if (nullptr == participant)
+        {
+            // Error
+            return;
+        }
+
+        // Optional: Set the preprocessor to be executed before parsing the IDL
+        DynamicTypeBuilderFactory::get_instance()->set_preprocessor("<optional_path_to_your_preprocessor_executable>");
+
+        // Load the IDL file
+        std::string idl_file = "<path_to_idl>.idl";
+        std::string type_name = "YourType";
+        std::vector<std::string> include_paths;
+        include_paths.push_back("<path_to_included_idl>.idl");
```

Comment:
I thought `include_paths` should contain paths instead of idl files. Can you check this?